modules/migration-migrating-applications-api.adoc

```diff
@@ -0,0 +1,311 @@
+<2> You must ensure that the secrets referenced in the `credsSecretRef` for the `backupStorageConfig` exist.
+<3> You must ensure that the secrets referenced in the `credsSecretRef` for the `volumeSnapshotConfig` exist.
+
+. Create the `MigStorage` instance:
++
+[source,terminal]
+----
+$ oc create -f migstorage.yaml
+----
+
+. Check the `MigStorage` instance to verify that it is in a `Ready` state:
++
+[source,terminal]
+----
+$ oc describe migstorage <migstorage_name>
+----
+
+. Create a `MigPlan` CR manifest called `migplan.yaml` on the target cluster:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigPlan
+metadata:
+  name: <migration-plan>
+  namespace: openshift-migration
+spec:
+  destMigClusterRef:
+    name: host
+    namespace: openshift-migration
+  indirectImageMigration: true <1>
+  indirectVolumeMigration: true <2>
+  migStorageRef:
+    name: <aws_s3> <3>
+    namespace: openshift-migration
+  namespaces:
+    - <namespace> <4>
+  persistentVolumes:
+    - capacity: 1Gi
+      name: <pvc-af0be31d-5b39-11eb-aceb-029d2246b63f> <5>
+      pvc:
+        accessModes:
+          - ReadWriteOnce
+        hasReference: true
+        name: <pvc_name> <6>
+        namespace: <namespace> <7>
+      selection:
+        action: <copy> <8>
+        copyMethod: <filesystem> <9>
+        storageClass: <gp2> <10>
+      storageClass: <glusterfs_storage> <11>
+      supported:
+        actions:
+          - copy
+          - move
+        copyMethods:
+          - filesystem
+          - snapshot
+  srcMigClusterRef:
+    name: <source_cluster> <12>
+    namespace: openshift-migration
+----
+<1> Change the value of `indirectImageMigration` to `false` for direct image migration.
+<2> Change the value of `indirectVolumeMigration` to `false` for direct volume migration.
+<3> Specify the name of the `MigStorage` instance.
+<4> Specify the namespace to be migrated.
+<5> Specify the PV.
+<6> Specify the persistent volume claim (PVC).
+<7> Specify the namespace to be migrated.
+<8> Specify `copy` or `move`.
+<9> Specify `filesystem` or `snapshot`.
+<10> Specify the source storage class.
+<11> Specify the target storage class.
+<12> Specify name of the source cluster `MigCluster` instance.
+
+. Create the `MigPlan` instance:
++
+[source,terminal]
+----
+$ oc create -f migplan.yaml
+----
+
+. Check the `MigPlan` instance to verify that it is in a `Ready` state:
++
+[source,terminal]
+----
+$ oc describe migplan <migplan_name>
+----
+
+. Create a `MigMigration` CR manifest for the source cluster called `migmigration.yaml`:
++
+[source,yaml]
+----
+apiVersion: migration.openshift.io/v1alpha1
+kind: MigMigration
+metadata:
+  name: <982ba500-5b3a-11eb-bd49-4734b4f50789>
+  namespace: openshift-migration
+spec:
+  migPlanRef:
+    name: <migplan_name>
+    namespace: openshift-migration
+  quiescePods: true
```

Comment:
Just a suggestion, can we add a numbered annotation for spec flags like `quiesePods` and `stage` for explaination/context purposes? Similar to what is done for other CR (MigPlan, MigStorage) spec fileds. 
`quiesePods:` Specifies whether to quiesce the application Pods before migrating Persistent Volume data.
`stage:` Invokes the stage operation, when set to true the migration controller switches to stage itinerary. This is a required field.